src/NAR/Constructs/SquareRootConstruct.java

```diff
@@ -25,8 +25,7 @@ public boolean isCorrectSyntax(String statement) {
     @Override
     public void setArgs(String statement) {
         String[] components = splitIntoComponents(statement.toLowerCase());
-        //TODO make sure components are not variables or parseInt will crash.
-        //Double[] vars=getVariables(components[0].substring(1),components[1].substring(1));
+
         String v1,v2;
         try {
             if (components[0].charAt(0) == '-') {
```

Comment:
you should be checking for the negative number in isCorrectSyntax, otherwise the user program will continue running even with this error